src/quotes/quotes.js

```diff
@@ -0,0 +1,148 @@
+import { FRONT_END_ROLE, BACK_END_ROLE, DEV_OPS_ROLE } from '../constants/roles';
+import { BRAG, CONFESS } from '../constants/roleActions';
+
+export const PAST = 'past';
+export const FUTURE = 'future';
+export const PREFIX = 'prefix';
+
+// ORDER HERE MATTERS, ALWAYS ADD AT THE END
+const PREFIX_LEAF = {
```

Comment:
I would consider moving it to different file **prefixes.js**

Reply:
those are collapsed into one structure, it seems to be more convenient for me "as initial writer" to have those together -> so I can easily optically match beginings and continuations of the senteces. Rather than switch between files, I would keep it for now